gatekeeper/gatekeeper.go

```diff
@@ -0,0 +1,60 @@
+// Package gatekeeper is a web service that will listen for a machine identity from a Cloud
+// provider, and request machine credentials.
+package gatekeeper
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli"
+
+	"github.com/manifoldco/torus-cli/api"
+	"github.com/manifoldco/torus-cli/config"
+	"github.com/manifoldco/torus-cli/gatekeeper/http"
+)
+
+// Gatekeeper provides the Listener interface for a daemonized HTTP co-process that will act
+// to register machines
+type Gatekeeper struct {
```

Comment:
It was used before in the co-process more since it had the `Listener` interface implemented, but now that it's a single process, it's not necessary.